client/src/App.js

```diff
@@ -417,4 +417,6 @@ class App extends Component {
 
 // please update your components and Routes as needed
 
-export default withAuthentication(App);
+// export default withAuthentication(App);
+
+export default App ;
```

Comment:
I think we need this to be withAuthentication but if it's not needed please remove the comment